unlock-js/src/__tests__/v11/partialWithdrawFromLock.test.js

```diff
@@ -31,7 +31,7 @@ describe('v11', () => {
 
       const callMethodData = prepContract({
         contract: UnlockV11.PublicLock,
-        functionName: 'partialWithdraw',
+        functionName: 'withdraw',
```

Comment:
`partialWithdraw` does not exist anymore on the new version since `withdraw` supports its logic.